### ISSUE TITLE
Variable names ending with `and` get split in `WHERE` statements

### ISSUE THREAD

**Issue by sambrilleman** [PabloRMira/sql_formatter#161]
### Describe the bug

Variable names ending with `and` get split in `WHERE` statements. E.g. variable name `brand` becomes `br and`.

### To Reproduce

```
sql_formatter.core.format_sql("""SELECT brand FROM table WHERE brand = 'my_brand'""")
Out[16]: "SELECT brand\nFROM   table\nWHERE  br\n   and = 'my_brand'"

sql_formatter.core.format_sql("""SELECT hello FROM table WHERE TRUE AND brand = 'my_brand'""")
Out[12]: "SELECT hello\nFROM   table\nWHERE  true\n   and br\n   and = 'my_brand'"
```

### Expected behavior

```
"SELECT hello\nFROM   table\nWHERE  true\n   and brand = 'my_brand'"
```

### Screenshots

N/A

**Comment by PabloRMira** [comment]
Many thanks for pointing out and for the very nice bug report! :-)

**Comment by sambrilleman** [comment]
Wow, thanks for the super fast fix! 😀

**Comment by PabloRMira** [comment]
You are welcome! :-)

Btw. I've just released the two fixes with version 0.6.2
